docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6
+
+RUN apt-get update
+RUN apt-get install -y ffmpeg mediainfo
+
+RUN wget $(curl --silent https://api.github.com/repos/DakaraProject/dakara-server/releases/latest | grep browser_download_url | sed -e 's/^.*\": \"//' -e 's/\"$//') -O dakara-server.zip
+
+RUN unzip dakara-server.zip
+RUN rm dakara-server.zip
+RUN mv dakara-server* /dakara-server
+RUN pip install -r /dakara-server/requirements.txt
+
+RUN cp /dakara-server/dakara_server/dakara_server/local_settings.py.example /dakara-server/dakara_server/dakara_server/local_settings.py
+RUN sed -i "s/^ALLOWED_HOSTS\ =\ \[/ALLOWED_HOSTS\ =\ \[\'\*\'/" /dakara-server/dakara_server/dakara_server/local_settings.py
```

Comment:
Just as a remark: the next release have a better settings management system.

Reply:
Noted, I don't mind waiting for the next release before merging this if you want.